tests/framework/resources.py

```diff
@@ -428,6 +428,74 @@ def create_json(
         return datax
 
 
+class SnapshotCreate():
+    """Facility for sending snapshot instructions on the microvm."""
```

Comment:
Can we differentiate this comment and the one from line 470 somehow?

Reply:
done